pkg/runner/writer_templates/server.go

```diff
@@ -0,0 +1,18 @@
+package writer_templates
+
+import (
+	"context"
+	"log"
+)
+
+type ReportServer struct {
+	UnimplementedReportWriterServer
+}
+
+func (s *ReportServer) SendReportResult(ctx context.Context, req *ReportResultRepeated) (*Empty, error) {
```

Comment:
Why do we need this server?

Reply:
this server is used for testing, when receiving the report, it will print it.